src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -42,35 +42,55 @@
 	public static final String UNKNOWN_VIDEO_TYPEMIME = "video/mpeg";
 	public static final String UNKNOWN_IMAGE_TYPEMIME = "image/jpeg";
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
+	public static final String AUDIO_TRANSCODE = "audio/transcode";
+	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AC3_TYPEMIME = "audio/vnd.dolby.dd-raw";
+	public static final String AUDIO_ADPCM_TYPEMIME = "audio/x-adpcm";
```

Comment:
As a side note for curious, and the ones with a weak memory ;-) Nothing to change.

audio/32kadpcm (IETF registred)